ansible_3par_docker_plugin/tasks/install_prerequisites_on_all.yml

```diff
@@ -40,6 +40,36 @@
       ('hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver'  in  my_driver_list and 'hpedockerplugin.hpe.hpe_3par_iscsi.HPE3PARISCSIDriver' in my_driver_list) or 
       ('hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver'  in  my_driver_list and 'hpedockerplugin.hpe.hpe_3par_fc.HPE3PARFCDriver' in  my_driver_list )    
      
+  - name: Fail when DEFAULT_FILE backend and DEFAULT_BLOCK is defined with single configuration 
+    fail:
+      msg: "DEFAULT_FILE and DEFAULT_BLOCK backends should not be present in single configuration"
+    with_items:
+      - "{{ INVENTORY.keys()}}" 
+    when: >-
+      set_mixed == false and 
+      (INVENTORY['DEFAULT_FILE'] is defined or INVENTORY['DEFAULT_BLOCK'] is defined)
+
+  - name: Fail when DEFAULT backend is having file driver and DEFAULT_FILE backend with file driver is defined 
+    fail:
+      msg: "DEFAULT_FILE backend should not be present when DEFAULT backend has filedriver"
+    with_items:
+      - "{{ INVENTORY.keys()}}" 
+    when: >-
+      set_mixed == true and 
+      INVENTORY['DEFAULT']['hpedockerplugin_driver'] == 'hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver' and 
+      INVENTORY['DEFAULT_FILE'] is defined
```

Comment:
Shouldn't we have a check for INVENTORY['DEFAULT_FILE']['hpedockerplugin_driver'] == 'hpedockerplugin.hpe.hpe_3par_file.HPE3PARFileDriver' here?

Reply:
Not required here since here we are verifying strictly on the keyword DEFAULT_FILE backend should not be present when you have file driver in DEFAULT 
the driver verification is covered in the L93 to L99 when we have DEFAULT_FILE defined and which does not have correct driver